src/code/main.c

```diff
@@ -117,6 +155,9 @@ void Main(void* arg) {
 
         osRecvMesg(&irqMgrMsgQueue, (OSMesg*)&msg, OS_MESG_BLOCK);
         if (msg == NULL) {
+#if PLATFORM_N64
+            if (1) {}
+#endif
```

Comment:
taking from MM, changing the following bit to
```c
        switch (*msg) {
            case OS_SC_PRE_NMI_MSG:
                PRINTF("main.c: リセットされたみたいだよ\n"); // "Looks like it's been reset"
                PreNmiBuff_SetReset(gAppNmiBufferPtr);
                break;
        }
```
removes the need for `if (1) {}` and matches all versions